programmers/비밀지도.js

```diff
@@ -0,0 +1,32 @@
+function solution(n, arr1, arr2) {
+    const passwords = [];
+    const results = [];
+    
+    let sum = 0;
+    let str = '';
+
+    for (let i = 0; i < arr1.length; i++) {
+        sum = arr1[i] | arr2[i];
+        sum = sum.toString(2);
+        passwords.push(sum)
+    }
+
+    for (let i = 0; i < passwords.length; i++) {
+        
+        for (let j = 0; j < passwords.length; j++) {
+            
+	    if (passwords[i][j] === '1') str += '#';
+            else if(passwords[i][j] === '0') str += ' ';
+        }
+
+        if (passwords[i].length !== n) {
+           
+            for (let k = 0; k < n - passwords[i].length; k++) {
+                str = ' ' + str;
+            }
+        }
+        results.push(str);
+        str = ''
+    }
+   return results;
+}
```

Comment:
변수명 선언해 줄때 들여쓰기는 따로 안하고 함께 작성해주는게 좋을거 같아요
그리고 불필요한 들여쓰기는 제거 해주시고 컨벤션 계속 신경 써 주세요!
명령문 마무리 할때 세미콜론(;)도 붙여주세요~ 
```suggestion
function solution(n, arr1, arr2) {
    const passwords = [];
    const results = [];
    let sum = 0;
    let str = '';

    for (let i = 0; i < arr1.length; i++) {
        sum = arr1[i] | arr2[i];
        sum = sum.toString(2);
        passwords.push(sum);
    }

    for (let i = 0; i < passwords.length; i++) {    
        for (let j = 0; j < passwords.length; j++) {            
	    if (passwords[i][j] === '1') str += '#';
            else if (passwords[i][j] === '0') str += ' ';
        }
        if (passwords[i].length !== n) {   
            for (let k = 0; k < n - passwords[i].length; k++) {
                str = ' ' + str;
            }
        }
        results.push(str);
        str = '';
    }
   return results;
}
```